packages/legacy-integration/.eslintrc.json

```diff
@@ -10,10 +10,11 @@
         {
             "files": ["*.spec.ts"],
             "rules": {
+                "@typescript-eslint/no-unsafe-assignment": "off",
```

Comment:
Why are we turning this rule off?

Reply:
Thanks for pointing out! 
I've added it due to the broken type in the payment-integration-service.mock.
Fixed it, removed rule where it was added, due to the type fixing.